src/pages/about.tsx

```diff
@@ -1,29 +1,38 @@
 import * as React from "react";
 import { Header, Container, Segment, Icon } from "semantic-ui-react";
+import { default as DefaultLayout } from "../components/Layout";
 
-export default () => {
+interface AboutPageProps {
+  location: {
+    pathname: string;
+  };
+}
+
+export default (props: AboutPageProps) => {
   return (
-    <Container>
-      <Segment vertical>
-        <Header as="h2">
-          <Icon name="info circle" />
-          <Header.Content>
-            About
-          </Header.Content>
-        </Header>
-      </Segment>
-      <Segment vertical>
-        <p>
-          This starter was created by @fabien0102.
-        </p>
-        <p>
-          For any question, I'm on <a href="https://discord.gg/2bz8EzW" target="blank">discord #reactiflux/gatsby</a>
-        </p>
-        <p>
-          For any issues, any PR are welcoming
-          <a href="https://github.com/fabien0102/gatsby-starter/issues" target="blank"> on this repository</a>
-        </p>
-      </Segment>
-    </Container>
+    <DefaultLayout location={props.location}>
```

Comment:
Why do we need to encapsulate every pages inside this `DefaultLayout`?

Reply:
I'm thinking about an HOC called `withLayout` to be more flexible to in this case we will have:

```ts
const AboutPage = () => <div />;
export default withLayout(AboutPage);
```

So we don't need to repeat this `location` props everytime :wink:

Reply:
I think it makes sense. I will try it with HOC.